cwf/gateway/docker/c/Dockerfile

```diff
@@ -82,78 +65,66 @@ RUN apt-get -y update && apt-get -y install \
   libboost-regex-dev
 
 ENV MAGMA_ROOT /magma
-ENV C_BUILD /build/c
-ENV OAI_BUILD $C_BUILD/oai
-
-ENV CCACHE_DIR $MAGMA_ROOT/.cache/gateway/ccache
-ENV MAGMA_DEV_MODE 1
-ENV XDG_CACHE_HOME $MAGMA_ROOT/.cache
+ENV XDG_CACHE_HOME ${MAGMA_ROOT}/.cache
+
+# Copy Bazel files at root and third_party
+COPY WORKSPACE.bazel ${MAGMA_ROOT}/
```

Comment:
COPY typically adds a single layer for each statement. It appears all these files in this block are from the same local source going to the same destination. If you are not keen on individual layers for each change, you could consider combining them to a single COPY statement that reduces the layers and potentially the build time as well.

Reply:
Ah didn't realize you can copy multiple files in one step!